examples/window-app/common/src/ZclCallbacks.cpp

```diff
@@ -39,83 +40,19 @@ using namespace ::chip::app::Clusters::WindowCovering;
-    }
-    else if (attributePath.mClusterId == Id)
-    {
-        ChipLogProgress(Zcl, "Window  cluster ID: " ChipLogFormatMEI, ChipLogValueMEI(attributePath.mClusterId));
-    }
-    else
-    {
-        ChipLogProgress(Zcl, "Unknown cluster ID: " ChipLogFormatMEI, ChipLogValueMEI(attributePath.mClusterId));
-    }
-
-    WindowApp & app     = WindowApp::Instance();
-    EndpointId endpoint = attributePath.mEndpointId;
-
-    chip::app::DataModel::Nullable<chip::Percent100ths> current;
-    chip::app::DataModel::Nullable<chip::Percent100ths> target;
-    OperationalState opState;
-
-    switch (attributePath.mAttributeId)
-    {
-    case Attributes::Type::Id:
-        app.PostEvent(WindowApp::Event(WindowApp::EventId::CoverTypeChange, endpoint));
-        break;
-
-    case Attributes::CurrentPositionLiftPercent100ths::Id:
-        app.PostEvent(WindowApp::Event(WindowApp::EventId::LiftChanged, endpoint));
-        break;
-
-    case Attributes::CurrentPositionTiltPercent100ths::Id:
-        app.PostEvent(WindowApp::Event(WindowApp::EventId::TiltChanged, endpoint));
-        break;
-
-    case Attributes::TargetPositionLiftPercent100ths::Id:
-        Attributes::TargetPositionLiftPercent100ths::Get(endpoint, target);
-        Attributes::CurrentPositionLiftPercent100ths::Get(endpoint, current);
-        opState = ComputeOperationalState(target, current);
-        if (OperationalState::MovingDownOrClose == opState)
-        {
-            app.PostEvent(WindowApp::Event(WindowApp::EventId::LiftDown, endpoint));
-        }
-        else if (OperationalState::MovingUpOrOpen == opState)
-        {
-            app.PostEvent(WindowApp::Event(WindowApp::EventId::LiftUp, endpoint));
-        }
         break;
-
-    case Attributes::TargetPositionTiltPercent100ths::Id:
-        Attributes::TargetPositionTiltPercent100ths::Get(endpoint, target);
-        Attributes::CurrentPositionTiltPercent100ths::Get(endpoint, current);
-        opState = ComputeOperationalState(target, current);
-        if (OperationalState::MovingDownOrClose == opState)
-        {
-            app.PostEvent(WindowApp::Event(WindowApp::EventId::TiltDown, endpoint));
-        }
-        else if (OperationalState::MovingUpOrOpen == opState)
-        {
-            app.PostEvent(WindowApp::Event(WindowApp::EventId::TiltUp, endpoint));
-        }
-        break;
-
     default:
         break;
     }
 }
 
-/**
- * @brief  Cluster StopMotion Command callback (from client)
- */
-bool emberAfWindowCoveringClusterStopMotionCallback(chip::app::CommandHandler * commandObj,
-                                                    const chip::app::ConcreteCommandPath & commandPath,
-                                                    const Commands::StopMotion::DecodableType & commandData)
+/* Forwards all attributes changes */
+void MatterWindowCoveringClusterServerAttributeChangedCallback(const app::ConcreteAttributePath & attributePath)
```

Comment:
I guess we can do this for now, but really this should be using the _app_ post-attribute-change callback, to trigger app logic, instead of overriding the cluster one.